### DISCUSSION TITLE
阿里云 获取app客户端可用token方法

### DISCUSSION THREAD

**Discussion by lalasou** [AlistGo/alist#1470] [Ideas]
抓包任意阿里云请求如图字段 base64解码
https://base64.us/

![image](https://user-images.githubusercontent.com/7440313/183439702-216ca238-be9b-41f4-a694-8605714e624a.jpeg)

如图
![image](https://user-images.githubusercontent.com/7440313/183439887-1d72d895-153f-4cbd-ac11-388eb49cf5ed.jpeg)

这个就是可用token

**Comment by rakaki** [comment]
抓不到。安装端，root装了根证书，还是无网络
